pkg/db/models/pr_commenting.go

```diff
@@ -11,12 +11,14 @@ const (
 )
 
 type PullRequestComment struct {
-	CreatedAt   time.Time
-	UpdatedAt   time.Time
-	PullNumber  int    `json:"pullNumber" gorm:"primaryKey"`
-	CommentType int8   `json:"commentType" gorm:"primaryKey"`
-	SHA         string `json:"sha" gorm:"primaryKey"`
-	Org         string `json:"org" gorm:"primaryKey"`
-	Repo        string `json:"repo" gorm:"primaryKey"`
-	ProwJobRoot string `json:"prowJobRoot"`
+	CreatedAt             time.Time
+	UpdatedAt             time.Time
+	PullNumber            int       `json:"pullNumber" gorm:"primaryKey"`
+	CommentType           int8      `json:"commentType" gorm:"primaryKey"`
+	SHA                   string    `json:"sha" gorm:"primaryKey"`
+	Org                   string    `json:"org" gorm:"primaryKey"`
+	Repo                  string    `json:"repo" gorm:"primaryKey"`
+	ProwJobRoot           string    `json:"prowJobRoot"`
+	LastCommentAttempt    time.Time `json:"lastCommentAttempt"`
+	FailedCommentAttempts int       `json:"failedCommentAttempts"`
```

Comment:
postgres/db-migration noob question: I see these are new.  How do those get added? is there some db-migration that just happens automatically?

Reply:
Yep, I know you found it already but for documentation purposes..

https://github.com/openshift/sippy/blob/4fca373f93a9e0b89585744ce3f5cd61ac11eb73/pkg/db/db.go#L128

Reply:
Yea there's some gorm magic that does it when we call AutoMigrate on the models

https://github.com/openshift/sippy/blob/master/pkg/db/db.go#L66